app/views/components/InputHidden.scala.html

```diff
@@ -14,12 +14,10 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.govukfrontend.views.Implicits.RichOptionString
 
-@this(govukInput: GovukInput)
+@this()
 
 @(field: Field)(implicit messages: Messages)
 
-@govukInput(
-  Input(inputType = "hidden").withFormField(field)
-)
+<input type="hidden" name="@{field.name}" id="@{field.id}"@field.value.mapNonEmpty { v => value=@v }>
```

Comment:
Would it ever make sense not to provide a value attribute? It's possibly not valid HTML.

id - not sure we need this. We provide ids for other inputs so they can be linked to from the error summary but this is non-visible.